### PR TITLE
Composite collision behaviour

### PR DESCRIPTION
**Pull request by daanwerf**
Also includes the gameplay balances which I have merged into this branch already

### CODE REVIEW

**Comment by SytzeAndr** [comment]
I approve.

**Comment by Exclaminator** [comment]
This branch does'nt include  composite pattern yet.

**Comment by daanwerf** [comment]
Then i don't understand what a composite pattern is. Could you elaborate?

File: src/main/java/lasersharks/behaviour/collision/AbstractCollisionBehaviour.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package lasersharks.behaviour.collision;
```
Comment (Exclaminator) [inline src/main/java/lasersharks/behaviour/collision/AbstractCollisionBehaviour.java]: What does this class add, please remove it.

**Pull request closed without merging**